spec/features/logging/view_auth_requests_for_a_username_spec.rb

```diff
@@ -2,10 +2,14 @@
   context "with results" do
     let(:username) { "AAAAAA" }
     let(:search_string) { "AAAAAA" }
+
     let(:organisation) { create(:organisation) }
     let(:admin_user) { create(:user, organisation_id: organisation.id) }
     let(:location) { create(:location, organisation_id: organisation.id) }
 
+    let(:organisation_with_no_ips) { create(:organisation) }
+    let(:admin_user_2) { create(:user, organisation_id: organisation_with_no_ips.id) }
```

Comment:
`organisation_id: organisation_with_no_ips.id)` should be `organisation: organisation_with_no_ips)`